content/courses/mobile/intro-to-solana-mobile.md

```diff
@@ -132,297 +124,320 @@ Unreal Engine.
+signing transactions.
+
+We will use the **Anchor framework** to interact with the on-chain counter
+program. The client side has already been developed in one of our previous
+lessons called
+[Intro to client-side Anchor development](https://solana.com/developers/courses/onchain-development/intro-to-anchor-frontend),
+feel free to check out its code for more context.
+
+To ensure you fully understand the core concepts, we will write this application
+in vanilla React Native without a starting template. While Solana Mobile offers
+templates that handle some boilerplate, building from scratch provides a much
+deeper understanding.
+
+### Getting Started
+
+To get started, you will need to properly set up a React Native development
+environment if you didn't already. This
+[article](https://reactnative.dev/docs/set-up-your-environment) shows you how.
+Remember that this step is not required if you are using a
+[Framework](https://reactnative.dev/architecture/glossary#react-native-framework).
+
+Ensure you have Node.js, npm, or yarn installed on your system. These will
+manage your JavaScript packages. Install Android Studio:
+
+Android Studio is required to run the Android emulator and to compile your React
+Native app for Android devices. Configure the ANDROID_HOME Environment Variable:
+
+> **NOTE:** You will need to configure the `ANDROID_HOME` environment variable
+> so that your terminal can recognize Android’s SDK tools. This step is critical
+> for running and building your app on Android.
+
+## Project Setup
+
+Create a Sample Project for the Emulator Setup to ensure your Android
+environment is set up correctly. In your terminal, run the code below within
+your preferred directory to scaffold a new React Native project, where
+`SampleProject` is your preferred project name. You can open the project in
+Android Studio and ensure it runs correctly on the Android emulator.
+
+```bash
+  npx react-native init SampleProject --npm
+```
+
+### Cloning and Running MWA
+
+1.  Clone the repo in `SampleProject`
+
+    ```bash
+    git clone https://github.com/solana-mobile/mobile-wallet-adapter.git
+    ```
+
+2.  In Android Studio, _Open project > Navigate to the cloned directory > Select
+    mobile-wallet-adapter/android_
+3.  After Android Studio finishes loading the project, select `fakewallet` in
+    the build/run configuration dropdown in the top right
+
+    ![Fake Wallet](/public/assets/courses/unboxed/basic-solana-mobile-fake-wallet.png)
+
+4.  For easier debugging, use **Logcat**. Check the
+    [official installation guide](https://developer.android.com/studio/debug/logcat)
+    if you are interested.
+5.  Now that your fake wallet is running on the emulator, go to _View -> Tool
+    Windows -> Logcat_. This will open up a console logging out what’s happening
+    with fake wallet.
+
+6.  (Optional) Install other
+    [Solana wallets](https://solana.com/ecosystem/explore?categories=wallet) on
```

Comment:
Link is dead. We got rid of these pages to avoid giving recommendations or having to vet potential malware. Was not my choice!

Reply:
haha, i replaced iit with a search from google play store instead: https://play.google.com/store/search?q=solana%20wallet&c=apps
is that fine?